src/NRediSearch/Query.cs

```diff
@@ -317,9 +324,20 @@ public Query AddFilter(Filter f)
         /// </summary>
         /// <param name="fields">a list of TEXT fields in the schemas</param>
         /// <returns>the query object itself</returns>
-        public Query LimitFields(params string[] fields)
```

Comment:
this is a breaking change; any changes should preserve the existing API (it is OK to forward if needed)